.circleci/config.yml

```diff
@@ -103,8 +103,8 @@ jobs:
             sudo npm i -g serverless
             yarn install
       - run:
-          name: Add .env file
-          command: yarn create-dotenv
```

Comment:
Could you explain what's the difference here?
This script is meant to dump everything into a .env file.